src/OrchardCore.Modules/OrchardCore.Queries/Sql/SqlGrammar.cs

```diff
@@ -107,7 +107,7 @@ public SqlGrammar() : base(false)
             columnItem.Rule = columnSource + aliasOpt;
 
             columnSource.Rule = funCall | Id;
-            fromClauseOpt.Rule = Empty | FROM + aliaslist + joinChainOpt;
+            fromClauseOpt.Rule = Empty | FROM + aliaslist + joinChainOpt + joinChainOpt + joinChainOpt + joinChainOpt + joinChainOpt;
```

Comment:
@sebastienros  Probably 
`MakeStarRule(joinChainOpt,` Empty)

instead of repeating it will do the job. I will give it a try tomorrow

Reply:
Did you see my PR ?

Reply:
I didn't, but I saw it, so that is great, it is working